help.go

```diff
@@ -190,8 +190,9 @@ func cliArgContains(flagName string, args []string) bool {
 }
 
 func printFlagSuggestions(lastArg string, flags []Flag, writer io.Writer) {
-	cur := strings.TrimPrefix(lastArg, "-")
-	cur = strings.TrimPrefix(cur, "-")
+	// Trim the prefix twice to handle both "-short" and "--long" flags.
+	currentArg := strings.TrimPrefix(lastArg, "-")
+	currentArg = strings.TrimPrefix(currentArg, "-")
```

Comment:
https://pkg.go.dev/strings#TrimLeft

Reply:
```suggestion
	currentArg := strings.TrimLeft(lastArg, "-")
```

Just in case it isn't obvious for other readers, we only need to trim once with `strings.TrimLeft`.

```go
package main

import (
	"fmt"
	"strings"
)

func main() {
	shortFlag := "-short"
	longFlag := "--long"

	fmt.Println(strings.TrimLeft(shortFlag, "-")) // short
	fmt.Println(strings.TrimLeft(longFlag, "-"))  // long

	fmt.Println(strings.TrimPrefix(shortFlag, "-")) // short
	fmt.Println(strings.TrimPrefix(longFlag, "-"))  // -long
}
```

Reply:
thanks @Juneezee, I didn't understand it before:)